nrf_dm/doc/CHANGELOG.rst

```diff
@@ -9,6 +9,21 @@ Changelog
 
 All the notable changes to this project are documented in this file.
 
+main
+****
+
+All the notable changes added to the |NCS| main are documented in this section.
+
+Added
+=====
+* :c:func:`void nrf_dm_get_hopping_sequence(const nrf_dm_config_t * config, uint8_t hopping_sequence[NRF_DM_CHANNEL_MAP_LEN])`, to get the channel hopping sequence used during the measurement procedure.
+* :c:func:`uint32_t nrf_dm_get_duration_us(const nrf_dm_config_t * config)`, to get the duration of a measurement procedure in microseconds.
+
+Changes
+=======
+* Invalid results of :c:func:`nrf_dm_high_precision_calc` now returns `NaN`.
+* Increase the number of measurements with quality :c:enumerator:`NRF_DM_QUALITY_OK`.
```

Comment:
```suggestion
* The number of measurements can now be increases with the :c:enumerator:`NRF_DM_QUALITY_OK` quality.
```

Reply:
It's more like:
The number of measurements with quality c:enumerator:`NRF_DM_QUALITY_OK` was increased.

Reply:
Updated to @Tschet1  suggestion here